sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/KafkaBundleWithConfigIT.java

```diff
@@ -860,16 +780,6 @@ public void shouldSetConsumerNameCorrectlyWithConsumerConfig() {
     }
   }
 
-  @Test
-  public void shouldSetConsumerNameCorrectlyWithConsumerConfigFromYaml() {
-    try (KafkaConsumer<String, String> c1 =
-        kafkaBundle.createConsumer(
-            new StringDeserializer(), new StringDeserializer(), CONSUMER_1)) {
-
-      assertThat(KafkaHelper.getClientId(c1)).isEqualTo(CONSUMER_1 + "-0");
-    }
-  }
-
```

Comment:
Why is this gone?

Reply:
https://github.com/SDA-SE/sda-dropwizard-commons/blob/f35d324874fd14657ac0e7c897d54c0ce83016af/sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/KafkaBundleWithConfigIT.java#L275 and https://github.com/SDA-SE/sda-dropwizard-commons/blob/f35d324874fd14657ac0e7c897d54c0ce83016af/sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/KafkaBundleWithConfigIT.java#L864 are identical. One test less to execute is one test less which can fail.